Lib/types.py

```diff
@@ -60,7 +60,27 @@ def _m(self): pass
 
 # Provide a PEP 3115 compliant mechanism for class creation
 def new_class(name, bases=(), kwds=None, exec_body=None):
-    """Create a class object dynamically using the appropriate metaclass."""
+    """Create a class object dynamically using the appropriate metaclass.
+
+    Arguments:
+    name -- the new class name
+    bases -- sequence of parent classes (default ())
+    kwds -- mapping of keyword arguments to be supplied to parents' `__init_subclass__`
```

Comment:
It is not only passed to `__init_subclass__`, it will be passed also to `__prepare__` of the computed metaclass, to the metaclass call itself, and in future to any other place required by the class creation machinery. This sentence should be reformulated to something simpler but correct, like `mapping of keyword arguments in an equivalent class statement (see example below).`